<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graph_dialog</class>
 <widget class="QDialog" name="graph_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Графики tstlan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/icons/icons/graph_2.png</normaloff>:/icons/icons/graph_2.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="graph_parameters_button">
       <property name="text">
        <string>Параметры графика</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="graphs_combobox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="graph_dialog_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="parameters_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="update_pparameters_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Обновить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="auto_update_parameters_checkbox">
           <property name="text">
            <string>Обновлять автоматически</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="parameters_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>260</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>0</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>7</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>8</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>9</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>10</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>11</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>12</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Параметр</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Значение</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Количество точек</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>Xmin</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>Xmax</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>Ymin</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>Ymax</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>Yaverage</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>Xmax - Xmin</string>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>|Ymax - Ymin|/|Yaverage|*100/2, %</string>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>СКО</string>
          </property>
         </item>
         <item row="9" column="0">
          <property name="text">
           <string>СКО / |Yaverage| * 100, %</string>
          </property>
         </item>
         <item row="10" column="0">
          <property name="text">
           <string>Доверительный интервал 0,95, %</string>
          </property>
         </item>
         <item row="11" column="0">
          <property name="text">
           <string>Доверительный интервал 0,99, %</string>
          </property>
         </item>
         <item row="12" column="0">
          <property name="text">
           <string>Доверительный интервал 0,999, %</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <layout class="QHBoxLayout" name="chart_layout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
